docs/discover/search-sessions.asciidoc

```diff
@@ -0,0 +1,75 @@
+there are times that long-running queries are required.
+Consider a threat hunting scenario, where you need to search through years of data.
+
+If your query is running long, you can save your search session, which
+allows {kib} to continue processing your request in the
+background.  Save your search session from *Discover* or *Dashboard*,
+and when your session is complete, view and manage it in *Stack Management*.
+
+[role="screenshot"]
+image::images/search-session.png[Search Session indicator displaying the current state of the search, which you can click to stop or save a running Search Session ]
+
+Search sessions are built on top of async search. For more information,
+refer to {ref}/async-search-intro.html[long-running searches].
+
+[float]
+==== Requirements
+
+Search sessions are enabled by default in your `kibana.yml` file.  For additional
+settings, refer to <<search-session-settings-kb>>.
+
+* To save a session, you must have permissions for *Discover* and *Dashboard*,
+and the <<kibana-feature-privileges, search sessions subfeature>>.
+
+* To view and restore a saved session, you must have access to *Stack Management*.
+
+* Before saving a session, you must disable auto refresh from the time filter.
++
+[role="screenshot"]
+image::images/disable-time-filter.png[Time filter showing how to turn off auto refresh]
+
+[float]
+==== Example: Save and manage a search session
+
+You’re trying to understand a trend you see on a dashboard in {kib}. You
+need to look at several years of data, currently in cold storage,
+but you don’t have time to wait. You want {kib} to
+continue working in the background, so tomorrow morning you can get your coffee,
+open your browser, and pick up where you left off.
+
+. Load your dashboard.
++
+Your search session begins automatically. The icon after the dashboard title
+displays the current state of the search. A clock indicates the session is in progress.
+A checkmark indicates that the search session is complete.
+
+. To instruct {kib} to continue the search in the background, click the clock icon,
+and then click *Save session*.
++
+[role="screenshot"]
+image::images/search-session-awhile.png[Search Session indicator displaying the current state of the search, which you can click to stop or save a running Search Session ]
++
+. When the search is complete, click the check icon, and then click *Manage sessions*.
++
+[role="screenshot"]
+image::images/search-session-complete.png[Popup with link to go directly to Search Sessions in Stack Management]
++
+. In the *Search Sessions* view in *Stack Management*, use the edit menu to:
+** *Inspect* the queries and filters that makeup the session.
+** *Extend* the expiration of a completed session.
+** *Delete* a session.
++
+[role="screenshot"]
+image::images/search-sessions-menu.png[Search Sessions management view with actions for inspecting, extending, and deleting a session. ]
+
+. To change the search timeout, go to *Stack Management > Advanced Setting*
```

Comment:
Unfortunately the description of this option in `Advanced Settings` is wrong.

This setting was used during the early development of search sessions to allow users run long search sessions while **on screen**. Now that we can send a search session to the background, it's no longer needed. And it doesn't effect the feature described here in any way.

I created issue https://github.com/elastic/kibana/issues/94912 to track its deprecation and I don't think we should mention it here.